src/reducers/transactionReducer.js

```diff
@@ -94,28 +93,25 @@ export const initialState = {
   updatesComplete: true,
   signingKeys: [0, 0], // default 2 required signers
   spendingStep: SPEND_STEP_CREATE,
-};
+});
 
 function updateInputs(state, action) {
-  const inputsTotalSats = action.value
-    .map((input) => input.amountSats)
```

Comment:
This seemed unnecessary especially since I believe it results in the array being iterated over twice when once will do.